front/pages/poke/index.tsx

```diff
@@ -14,12 +31,90 @@ export const getServerSideProps = withSuperUserAuthRequirements<object>(
+  <>
+    <h1 className="mb-4 mt-8 text-2xl font-bold">{title}</h1>
+    <ul className="flex flex-wrap gap-4">
+      {workspaces.length === 0 && <p>No workspaces found.</p>}
+      {workspaces.map((ws) => (
+        <Link href={`/poke/${ws.sId}`} key={ws.id}>
+          <li className="border-material-100 s-w-[320px] rounded-lg border bg-white p-4 transition-colors duration-200 hover:bg-gray-100">
+            <h2 className="text-md flex-grow pb-2 font-bold">{ws.name}</h2>
+            <PokeTable>
+              <PokeTableBody>
+                <PokeTableRow>
+                  <PokeTableCell colSpan={3}>
+                    <label>
+                      Created: {moment(ws.createdAt).format("DD-MM-YYYY")}
+                    </label>
+                    {ws.upgradedAt && (
+                      <label>
+                        Upgraded: {moment(ws.upgradedAt).format("DD-MM-YYYY")}
+                      </label>
+                    )}
+                  </PokeTableCell>
+                </PokeTableRow>
+                <PokeTableRow>
+                  <PokeTableCell>
+                    {ws.adminEmail}{" "}
+                    {ws.workspaceDomain && (
+                      <label>({ws.workspaceDomain.domain})</label>
+                    )}
+                  </PokeTableCell>
+                  <PokeTableCell align="center">
+                    {ws.membersCount && ws.membersCount > 1 ? (
+                      <label>
+                        <Icon visual={UsersIcon} /> {ws.membersCount}
+                      </label>
+                    ) : (
+                      <label>
+                        <Icon visual={UsersIcon} /> {ws.membersCount}
+                      </label>
+                    )}
+                  </PokeTableCell>
+                  <PokeTableCell align="center">
+                    {ws.dataSourcesCount && ws.dataSourcesCount > 1 ? (
+                      <label>
+                        <Icon visual={CloudArrowLeftRightIcon} />{" "}
+                        {ws.dataSourcesCount}
+                      </label>
+                    ) : (
+                      <label>
+                        <Icon visual={CloudArrowLeftRightIcon} />{" "}
+                        {ws.dataSourcesCount}
+                      </label>
+                    )}
+                  </PokeTableCell>
+                </PokeTableRow>
+                <PokeTableRow>
+                  <PokeTableCell className="space-x-2" colSpan={3}>
+                    <label className="rounded bg-green-500 px-1 text-sm text-white">
+                      {ws.sId}
+                    </label>
+                    {ws.subscription && (
+                      <label
+                        className={`rounded px-1 text-sm text-gray-500 text-white ${isEntreprisePlan(ws.subscription.plan.code) ? "bg-red-500" : isProPlan(ws.subscription.plan.code) ? "bg-orange-500" : isOldFreePlan(ws.subscription.plan.code) ? "bg-gray-300" : "bg-blue-500"}`}
```

Comment:
We tend to avoid nested ternaries. Also we have a helper `className()` to chain styles in a conditional way.